### PR TITLE
Restaurants page

### PR DESCRIPTION
**Pull request by MiljkovicVeljko**

### CODE REVIEW

File: client-app/src/app/app.module.ts [hunk 1]
```diff
@@ -51,13 +56,17 @@ const SERVICES = [UserService, RestaurantService, FoodService];
   imports: [
     BrowserModule,
     AppRoutingModule,
+    BrowserAnimationsModule,
     SocialLoginModule,
     HttpClientModule,
     NoopAnimationsModule,
+    MatTooltipModule,
     MatExpansionModule,
+    MatTooltipModule,
```
Comment (abericExecom) [inline client-app/src/app/app.module.ts:65]: Duplicate MatTooltipModule

File: client-app/src/app/pages/home/home.component.html [hunk 2]
```diff
@@ -1,4 +1,5 @@
 <div class="container">
   <app-header></app-header>
+  <app-restaurants-overview></app-restaurants-overview>
```
Comment (abericExecom) [inline client-app/src/app/pages/home/home.component.html:3]: <app-restaurants-overview> component should be routed via <router-outlet>

File: client-app/src/app/pages/restaurants-overview/restaurants-overview.component.ts [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+        },
+        {
+          id: 6,
+          name: 'mala kari piletina',
+          price: 250,
+          restaurantId: 5,
+        },
+      ],
+    },
+  ];
```
Comment (abericExecom) [inline client-app/src/app/pages/restaurants-overview/restaurants-overview.component.ts:256]: You should mocks to shared/mock directory please :D

File: client-app/src/app/pages/restaurants-overview/restaurants-overview.component.ts [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+
+  constructor(private restaurantService: RestaurantService) {}
+
+  ngOnInit(): void {
+    this.sub = this.restaurantService.fetchRestaurants().subscribe(({ restaurants }) => {
+      console.log(restaurants);
+
```
Comment (abericExecom) [inline client-app/src/app/pages/restaurants-overview/restaurants-overview.component.ts:263]: console.log and new line should be removed

File: client-app/src/app/pages/restaurants-overview/restaurants-overview.component.scss [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+  width: 30px;
+  height: 30px;
+}
+
+.card-title {
+  border-radius: 15px 15px 0 0;
+}
+
+.card {
+  border-radius: 15px;
```
Comment (mkovacevicExecom) [inline client-app/src/app/pages/restaurants-overview/restaurants-overview.component.scss:14]: all px to rem

File: client-app/src/app/pages/restaurants-overview/restaurants-overview.component.scss [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+  font-size: 1.2rem;
+}
+
+.card-title {
+  background: $color-primary;
+  color: $white;
+}
+
+.card-footer {
+  background: #f8df76;
```
Comment (mkovacevicExecom) [inline client-app/src/app/pages/restaurants-overview/restaurants-overview.component.scss:29]: move to pallete file

File: client-app/src/app/pages/restaurants-overview/restaurants-overview.component.scss [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+    }
+
+    & > .disabled {
+      visibility: hidden;
+    }
+  }
+}
+
+.custom-tooltip {
+  background: yellowgreen;
```
Comment (mkovacevicExecom) [inline client-app/src/app/pages/restaurants-overview/restaurants-overview.component.scss:74]: move to pallete file

File: client-app/src/assets/styles/_mixins.scss [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+  height: 0rem;
+  width: 0rem;
+  position: relative;
+  left: $parrent;
+
+  & > span {
+    position: absolute;
+    bottom: -3rem;
+    left: $x;
+    font-size: 50px;
```
Comment (mkovacevicExecom) [inline client-app/src/assets/styles/_mixins.scss:11]: px to rem

File: client-app/src/assets/styles/_mixins.scss [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+    color: #ddd;
+
+    & > :hover {
+      background: #aaa;
```
Comment (mkovacevicExecom) [inline client-app/src/assets/styles/_mixins.scss:15]: move to pallete file

**Review by abericExecom — APPROVED** [review]

File: client-app/src/app/pages/restaurants-overview/restaurants-overview.component.ts [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+  customOptions: OwlOptions = CAROUSEL_OPTIONS;
+  dataRestaurants: IRestaurant[] = RESTAURANTS;
+  private sub: Subscription;
+
+  constructor(private restaurantService: RestaurantService) {}
+
+  ngOnInit(): void {
+    this.sub = this.restaurantService.fetchRestaurants().subscribe(({ restaurants }) => {
+      // this.dataRestaurants = restaurants; // @TODO: connect with API (check API data first)
+    });
```
Comment (BraneOpacic21) [inline client-app/src/app/pages/restaurants-overview/restaurants-overview.component.ts:25]: 1. If we aren't using our subscription, we should comment it out all the way.
2. On line 24 - add TODO on a line before `this.dataRestautrants = restaurants`
3. `dataRestaurants` could be named better, probably 😄 
@MiljkovicVeljko

**Review by BraneOpacic21 — COMMENTED** [review]
@MiljkovicVeljko introduce this change as an fix in f-fix/pr-NUMBER_OF_YOUR_PR

**Pull request merged**